ui/src/jobs/components/DeadlineDate.jsx

```diff
@@ -1,12 +1,19 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
+import moment from 'moment';
 import IconText from '../../common/components/IconText';
 
 class DeadlineDate extends Component {
   render() {
     const { deadlineDate } = this.props;
+    const formattedDeadlineDate = moment(deadlineDate).format('MMM D, YYYY');
```

Comment:
I'd move this to a ``static``

Reply:
what would be the advantage of this?

Reply:
just to be consistent, we usually use ``static`` for calculated params

Reply:
I wouldn't make it static in this case as that won't add any value and it is also quite small, just a date format

Reply:
It's just a suggestion, you did it already for small things like ``renderRank`` https://github.com/inspirehep/inspirehep/pull/212/files#diff-f1c078865507d0e537db58f068149762R9 I don't see why it's different

Reply:
It's different because renderRank needs to be passed to the InLineList